### ISSUE TITLE
Broken Tag Pages when not using FURLs

### ISSUE THREAD

**Issue by jpdevries** [modxcms/Articles#39]
Steps to reproduce:
- install a fresh version of MODX with default settings
- install Articles
- create an Article container, publish it
- create an Article, give it some tags, publish it
- view Article
- click a tag
- notice you are taken to a page like `index.php?id=2/tags/pork` which serves Resource 1 (rather than 2) and does not list the appropriate tags

**Comment by KHysiek** [comment]
I've checked my code and I have fixed this also (when working on #38) issue.
Bug is located in Chunk "sample.ArticleRowTpl", line #3, original line (from my backup, still using Articles 1.6.3):

... &useTagsFurl=`1` ....  settingis hardcoded. I've fixed it this way:

 &useTagsFurl=`[[++friendly_urls]]`

Treat this as hint, not as foolproof solution :) due to my low experience.

BTW. I think there can be many errors in Articles' template and it's chunks, it looked for me as rather low quality and not up to moderna standards (IMHO as not too experienced HTML/PHP programmer). I saw someone volunteered in forums to do new template and I think it's a good idea to use his help and create template for Articles 2.0 from scratch.
